wordpress-mobile/WordPress-iOS
# Issue/8172 site creation site type

Fixes #8172 

**To test:**
- Go to Switch Sites.
- Tap the "+" button to create a new site.
- Select "Create a WordPress.com site".
- Verify the select site type page displayed as looks like the design.

**Notes:**
- The design has a Back button. However, since this is the first page, there is nothing to go back to. So a Cancel button is displayed instead.
- The Help button will crash the app until #8199 is fixed.
- When a Site Type is selected, this temporary alert appears:
<img width="125" alt="alert" src="https://user-images.githubusercontent.com/1816888/33084195-8d9f4370-ce9e-11e7-95f4-54377c187878.png">


**Implemented view:**

![select_site_type](https://user-images.githubusercontent.com/1816888/33045120-fd25df38-ce08-11e7-8acf-b4ccb43daa68.png)

**Designed view:**

<img width="200" alt="designed_view" src="https://user-images.githubusercontent.com/1816888/33045173-30b8ac90-ce09-11e7-9bba-8b8f3c55aa9b.png">
